superset-frontend/src/explore/components/ExploreViewContainer.jsx

```diff
@@ -404,7 +403,11 @@ function ExploreViewContainer(props) {
           dashboardId={props.dashboardId}
         />
       )}
-      <div
+      <Resizable
+        defaultSize={{ width: 320 }}
+        minWidth={320}
```

Comment:
I think the minWidth/default width for the Dataset column since it somehow looks wider visually comparing to the Controls column, probably because of the gray background or more white space.

Reply:
I think 300px for gray column looks ok. WDYT?
![image](https://user-images.githubusercontent.com/15073128/104302147-ba41d100-54c8-11eb-95c6-4dafb88c44a6.png)

Reply:
Looking good! Thanks for the update!